### ISSUE TITLE
Train srgan system with grayscale images (1 channel srgan)

### ISSUE THREAD

**Issue by anonymous13771998** [eriklindernoren/PyTorch-GAN#139]
Hi
I want to change the srgan program from 3 channels to 1 channel and do the train with grayscale images
Does anyone know?
Please help me

**Comment by lusc1996** [comment]
`train_loader=datasets.ImageFolder("dataset",
                                transform=transforms.Compose([
                                    transforms.Resize(opt.img_size),
                                    **transforms.Grayscale(1),**
                                    transforms.ToTensor(),
                                    transforms.Normalize([0.5], [0.5])]),
                                )`
May be you need to add transfors.Grayscale(1)
